docs-website/sidebars.js

```diff
@@ -142,7 +111,12 @@ module.exports = {
           ],
         },
         {
-          Sinks: list_ids_in_directory("metadata-ingestion/sink_docs"),
+          Sinks: [
+            {
+              type: "autogenerated",
+              dirName: "metadata-ingestion/sink_docs",
```

Comment:
How come these are in a different place from the source docs?
```
dirName: "docs/generated/ingestion/sources"
```

Reply:
The source docs are generated by another build script, whereas these are raw md files